multidim-interop/rust/manifest.toml

```diff
@@ -0,0 +1,19 @@
+name = "compatibility-rust"
```

Comment:
I wonder whether Testground still needs these. I was operating under the assumption that these are only needed to test the Rust version individually. With the DSL abstraction, I don't think this is needed, right?

Reply:
Yup, vestige of copying this code.